docs/Tutorials/1_Getting_Started/Getting started.md

```diff
@@ -0,0 +1,207 @@
+  createCmd Account.Factory
+    with
+      provider = bank
+      observers = empty
+```
+
+Notice how the `ContractId` is immediately cast to an interface upon creation: this is because our workflows do not have any knowledge of concrete template implementations.
+
+Similarly, we define a holding factory which is used within an account to `Credit` and `Debit` holdings.
+
+```Haskell
+holdingFactoryCid <- toInterfaceContractId @Holding.F <$> submit bank do
+  createCmd Fungible.Factory
+    with
+      provider = bank
+      observers = empty
+```
+
+This factory contract can be use to create `Fungible` holdings, which are defined in `Daml.Finance.Asset.Fungible` and are [fungible](../../Glossary.md#fungibility), as well as [transferable](../../Glossary.md#transferability).
+
+### Opening Alice's and Bob's accounts
+
+Once the factory templates are setup, we leverage our `CreateAccount` workflow to create an account at the `Bank` for `Alice` and `Bob`.
+
+The creation of an account needs to be authorized by both the `custodian` and the `owner` (resp. the `Bank` and `Alice` in our case). Authorization is collected using an initiate / accept pattern.
+
+```Haskell
+aliceRequestCid <- submit alice do
+  createCmd AccountOpenRequest
+    with
+      owner = alice
+      custodian = bank
+
+aliceAccountCid <- submit bank do
+  exerciseCmd aliceRequestCid AccountOpenRequest_Accept
+    with
+      label = "Alice@Bank"
+      accountFactoryCid = accountFactoryCid
+      holdingFactoryCid = holdingFactoryCid
+      observers = []
+```
+
+`Bob`'s account is created in a similar fashion.
+
+### Issuing the cash instrument
+
+In order to credit Alice's account with some cash, we first need to introduce a cash `Instrument` in our model.
+
+```Haskell
+let instrumentId = Id with label = "USD"; version = "0"
+
+cashInstrumentCid <- toInterfaceContractId @Instrument.I <$> submit bank do
+  createCmd Instrument
+    with
+      depository = bank
+      issuer = bank
+      id = instrumentId
+      observers = empty
+```
+
+An instrument is a representation of what it is that we are holding against the bank. It can be as simple as just a textual label (like in this case) or can include complex on-ledger lifecycling logic.
+
+To hold one unit of the cash instrument in this scenario means that we can claim 1 USD from the holding's custodian.
+
+Notice how in this case the `Bank` acts both as the cash instrument's issuer and depository. This means that we fully trust the `Bank` with any action concerning the instrument.
+
+### Depositing cash on Alice's account
+
+We can now deposit cash on Alice's account, using the `Deposit` workflow. Alice issues a deposit
+
+```Haskell
+aliceRequestCid <- submit alice do
+  createCmd DepositRequest
+    with
+      account = AccountKey with owner = alice; custodian = bank; id = "Alice@Bank"
+      instrument = InstrumentKey with issuer = bank; depository = bank; id = instrumentId
```

Comment:
We should explain how/where these keys are derived from.

Reply:
Agree. Do you think this is best done in the tutorial or in the (to-be-created) documentation?

Reply:
i was thinking of Main Documentation > Core Concepts > explain what an Account (resp. Instrument) is and how it is keyed

Reply:
I think its fine if we mention that this will either be explained later or a link to where it is explained.